mod_reforged/hooks/config/strings.nut

```diff
@@ -293,7 +293,7 @@ local vanillaDescriptions = [
 	 		Effects = [{
  				Type = ::UPD.EffectType.Active,
  				Description = [
-					"Unlocks the [Adrenaline|Skill+adrenaline_skill] skill which puts you first in the [turn|Concept.Turn] order for the next [round|Concept.Round], to have another [turn|Concept.Turn] before your enemies do."
+					"Unlocks the [Adrenaline|Skill+adrenaline_skill] skill which puts you first in the [turn|Concept.Turn] order for the next [round|Concept.Round], to have another [turn|Concept.Turn] before your enemies do and ignore [temporary injuries|Concept.InjuryTemporary]."
```

Comment:
We need to some how also convey the information here that it makes you immune to receiving injuries. Either that has to be done explicitly in words, or via a nested tooltip of the adrenaline effect here. So we could do:
```squirrel
"Unlocks the [Adrenaline|Skill+adrenaline_skill] skill which puts you first in the [turn|Concept.Turn] order for the next [round|Concept.Round], to have another [turn|Concept.Turn] before your enemies do.",
"Under the effects of [adrenaline|Skill+adrenaline_effect] you cannot receive and are not affected by [temporary injuries|Concept.InjuryTemporary]."
```
Or
```squirrel
"Unlocks the [Adrenaline|Skill+adrenaline_skill] skill which puts you first in the [turn|Concept.Turn] order for the next [round|Concept.Round], and grants the [Adrenaline|Skill+adrenaline_effect] effect which makes you immune to [injuries|Concept.InjuryTemporary].",
```